content/en/releases/patch-releases.md

```diff
@@ -86,9 +78,10 @@ releases may also occur in between these.
 
 | Monthly Patch Release | Target date |
 | --- | --- |
-| May 2021 | 2021-05-12 |
 | June 2021 | 2021-06-16 |
 | July 2021 | 2021-07-14 |
+| August 2021 | 2021-08-11 |
+| September 2021 | 2021-09-15 |
```

Comment:
Do we have some rules that we're following for choosing the release day? I'm mostly wondering about August, since Wednesday after the 11th is the 18th, which seems like a reasonable release date as well. IIRC, we did releases on and around the 18th of the month in the past.

Reply:
I don't think those rules would be part of this PR.

Reply:
> I don't think those rules would be part of this PR.

Agreed.
@xmudrii -- would you like to own discussing a policy as a follow-up action?

Reply:
@justaugustus -- Sounds good. I'll take care of that.